akka-cluster-tools/src/main/scala/akka/cluster/singleton/ClusterSingletonManager.scala

```diff
@@ -44,7 +44,7 @@ object ClusterSingletonManagerSettings {
    */
   def apply(system: ActorSystem): ClusterSingletonManagerSettings =
     apply(system.settings.config.getConfig("akka.cluster.singleton"))
-      .withRemovalMargin(Cluster(system).settings.DownRemovalMargin)
+      .withRemovalMargin(Cluster(system).downingProvider.downRemovalMargin)
```

Comment:
I don't think it's as bad as it looks. See `val removalMargin` in `ClusterSingletonManager` class.
It means that by default the `downingProvider.downRemovalMargin` is used. It's only if `akka.cluster.down-removal-margin != off` it will use that value instead. It was probably to keep some compatibility.

I'm thinking that perhaps we should remove the deprecation of `akka.cluster.down-removal-margin`.
After all, it's a different thing than `stable-after` so it might make sense to make it possible to configure that separately.

By default, in SBR we can use the `stable-after` value if `down-removal-margin = off`.

Reply:
Trixy, I didn't see or remember that.

Ok, so we: 
1. keep this parameter as is
2. remove the deprecation for the config file setting (but maybe clarify the doc-comment there)
3. update the SBR docs to only mention the other setting (and possibly that it can be overriden by changing this)
4. profit

?